data/Act.csv

```diff
@@ -1243,3 +1243,22 @@ A01667,P1030,AT01,PS00842,,1950..1970~,1950..1970~,,as part of her morning routi
 A01668,P0999,AT55,PS09999,,1950..1970~,1950..1970~,,read out at a meeting,SS00600,546,3,F,unknown work G55,check reading time and narrative time,2020-02-13,MG,2020-02-13,MG
 A01669,P1056,AT13,PS00850,,1950..1970~,1950..1970~,,,SS00600,547,3,F,,check reading time and narrative time,2020-02-13,MG,2020-02-13,MG
 A01670,P1056,AT17,PS00850,,1950..1970~,1950..1970~,,,SS00600,547,3,F,,check reading time and narrative time,2020-02-13,MG,2020-02-13,MG
+A01676,P1061,AT05,PS00087,,1967,1967,,This is while he listens to his wife who preaches him about how bad his class background is. He thinks to himself that his wife is not even allowed herself to read the Little Red Book.,SS00601,11,3,F,,,2020-02-24,MG,2020-02-24,MG
+A01677,P1060,AT11,PS00857,,1967~,1967~,,Bai is entrusted with translating the part about the trees and forests (so he says).,SS00601,23,3,F,,pls check act_start and end,2020-02-24,MG,2020-02-24,MG
+A01678,P1060,AT03,PS00857,P1059,1967~,1967~,,he lets her keep the book for now but tells her to be cautious.,SS00601,23,3,F,,pls check act_start and end,2020-02-24,MG,2020-02-24,MG
+A01679,P1060,AT19,PS00857,P1059,1967~,1967~,,"Bai and Ye are both appalled at the pointless environmental destruction that is taking place in the countryside with the high-speed cutting down of old trees.
+Bai also explains that the book was published in 1962 in the West and become very influential. Bai wants to tell the Beijing leadership about the destruction.",SS00601,23,3,F,,pls check act_start and end,2020-02-24,MG,2020-02-24,MG
+A01680,I0008,AT06,PS00857,,1967~,1967~,,The book is being prepared for internal publication,SS00601,23,3,F,,pls check act_start and end,2020-02-24,MG,2020-02-24,MG
```

Comment:
this is the first institutional agent, there will probably be more, each requires manual update of the bats test for now.